src/Pages/GlobalConfigurations/DeploymentCharts/types.ts

```diff
@@ -16,30 +16,34 @@
 
 import { ResponseType } from '@devtron-labs/devtron-fe-common-lib'
 
-export interface ChartDetailType {
-    id: number
-    chartDescription: string
+// TODO: move this type to fe-common-lib since can be reused in fe-lib
```

Comment:
Are we going to add this in tech debt?

Reply:
moving it to common-lib into a new folder in Pages > Global Configuration > Deployment Charts